### PR TITLE
Adding web documentation for Twist miniapp [twist-dev]

### PR DESCRIPTION
**Pull request by mlstowell**
This is a companion PR to [PR#1306](https://github.com/mfem/mfem/pull/1306).

### CODE REVIEW

File: src/examples.md [hunk 1]
```diff
@@ -1303,6 +1336,7 @@ function update(id)
    showElement("mobius-strip", meshing && all2 && all3 && all4);
```
Comment (tzanio) [inline src/examples.md]: Can you increment the number of meshing miniapps above?

File: src/examples.md [hunk 2]
```diff
@@ -995,6 +995,39 @@ moving to the miniapps.**_
...
+defining a stack of individual elements and stitching together the top and
+bottom of the mesh. The stack can also be twisted so that the vertices of the
+bottom and top can be joined with any integer offset (for tetrahedral and
+wedge meshes only even offsets are supported).
+
+The Twist miniapp supports various options including:
+
+- The element type: 4 - Tetrahedron, 6 - Wedge, 8 - Hexahedron
+- The geometric order of the elements
+- The dimensions of the inital brick-shaped stack of elements
```
Comment (tzanio) [inline src/examples.md]:
```suggestion
- The dimensions of the initial brick-shaped stack of elements
```

File: src/examples.md (hunk 1 above)
Comment (mlstowell) [reply src/examples.md]: Of course!

**Comment by tzanio** [comment]
@v-dobrev, can you deploy?

**Comment by v-dobrev** [comment]
> @v-dobrev, can you deploy?

Done.

**Comment by tzanio** [comment]
Thank you 🙇

**Pull request merged**